static/docs/command-reference/repro.md

```diff
@@ -45,6 +45,10 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Parallel execution
+
+See [Parallel execution](/doc/command-reference/run#Parallel-execution).
```

Comment:
is it enough? it's not clear why is it the same, tbh. Some clarification is required on why this link is relevant in explaining this.